inc/usb_std.h

```diff
@@ -126,6 +126,7 @@ extern "C" {
 #define USB_DTYPE_OTG               0x09    /**<\brief OTG descriptor.*/
 #define USB_DTYPE_DEBUG             0x0A    /**<\brief Debug descriptor.*/
 #define USB_DTYPE_INTERFASEASSOC    0x0B    /**<\brief Interface association descriptor.*/
+#define USB_DTYPE_FUNCTIONAL_DESC   0x21   /**<\brief Functonal descriptor.*/
```

Comment:
Same definition found in usb_ccid.h. Is this descriptor generic or CCID-related only?

Reply:
Looks like 0x21 code also used as a functional descriptor in DFU. So it may be generic, but this is not specified. For me the best solution to rename USB_DTYPE_FUNCTIONAL_DESC to USB_DTYPE_CCID_FUNCTIONAL and leave it in class-specific header.